orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -197,7 +197,7 @@ public Stage preceding(String type) {
    * Maps the stage's context to a typed object at a provided pointer. Uses
    * <a href="https://tools.ietf.org/html/rfc6901">JSON Pointer</a> notation for determining the pointer's position
    */
-  <O> O mapTo(String pointer, Class<O> type) {
+  public <O> O mapTo(String pointer, Class<O> type) {
```

Comment:
@robfletcher We have a number of callers to this ... they were still working "because groovy"